fbw-a32nx/src/systems/instruments/src/EFB/Efb.tsx

```diff
@@ -291,6 +299,13 @@ const Efb = () => {
     // </Pushback>
     // =========================================================================
 
+    // Writes the initial values from the global settings.
+    const syncGlobalSettings = () => {
+       dispatch(setPassengerAmbienceActive(passengerAmbienceEnabled === 1 ? true : false));
+       dispatch(setCabinAnnouncementsActive(cabinAnnouncementsEnabled === 1 ? true : false));
+       dispatch(setBoardingMusicActive(boardingMusicEnabled === 1 ? true : false));
```

Comment:
You actually don't need to supply `true` or `false` in a ternary statement here, since that is the result of the conditional.

So you can simplify this down to:
```suggestion
       dispatch(setPassengerAmbienceActive(passengerAmbienceEnabled === 1));
       dispatch(setCabinAnnouncementsActive(cabinAnnouncementsEnabled === 1));
       dispatch(setBoardingMusicActive(boardingMusicEnabled === 1));
```

Reply:
Yes thats correct, i´ll add this !